File: example_code/mpi/MPI_Code/C/hello.c
#include <stdio.h>
#include "mpi.h"

int main(int argc, char* argv[])
{
   /* Initialize the library */
   MPI_Init(&argc, &argv);

   /* Parallel Code */
   printf("Hello world\n");

   /* Warp it up. */
   MPI_Finalize();
   return 0;
}

File: example_code/mpi/MPI_Code/Fortran/hello.f90
PROGRAM hello
!USE mpi_f08
!USE mpi
INCLUDE 'mpif.h'
INTEGER::IERR
! Initialize the library
CALL MPI_Init(IERR)

PRINT*,'Hello World'

! Warp it up.
CALL MPI_Finalize(IERR)
END PROGRAM hello
